test/types/community/collection/replaceX.test-d.ts

```diff
@@ -0,0 +1,22 @@
+import { expectError } from 'tsd';
+import { MongoClient, ObjectId } from '../../../../src';
+
+// test collection.replaceX functions
+const client = new MongoClient('');
+const db = client.db('test');
+
+interface TestModel {
+  _id: ObjectId;
+  stringField: string;
+}
```

Comment:
Can basically take a clone of this schema, except try a Model that has an _id that is a custom type like `_id: number`

Reply:
@nbbeeken Just wanted to follow up on this - could you clarify what you mean? Were you asking for a new test case?

Reply:
Yea my comment is no longer relevant, when using `OptionalId` we'd have to test for `ObjectId` type and non-`ObjectId` type. But `WithoutId` omits the `_id` altogether so we don’t need the dupe test for other types.